pandas/plotting/_core.py

```diff
@@ -3486,6 +3595,12 @@ def scatter(self, x, y, s=None, c=None, **kwds):
               recursively. For instance, when passing [2,14] all points size
               will be either 2 or 14, alternatively.
 
+            - .. versionadded:: 0.24.0
```

Comment:
I'm not sure how sphinx will handle this... I would say just do

```
s : int, str, scalar or array_like, optional
    The size of each point....
    ...
    - a column name containing numeric or ordered categorical data...
    
    .. versionchanged:: 0.24.0
       `s` can now be a column name. 
```

Reply:
+1